docs/source/index.rst

```diff
@@ -52,13 +52,16 @@ that approach may not be as up to date.
 .. toctree::
    :hidden:
    :maxdepth: 1
+   :titlesonly:
 
    user-guide/index
    configuration
    api/index
```

Comment:
This "Conda Python API" is overlapping with the auto-generated API docs, let's move this together with the autoapi stuff into a new `dev-guide/api` section (with an own index document), in which can link to both the "official API" and the autogenerated one

Reply:
We'll need to set up a redirect in readthedocs as well, once we've merged this.

That reminds me, one of the main issues of porting the docs away from ReadTheDocs, is that all of the redirects are either implemented as weird files in the docs themselves, or in ReadTheDocs as server-side configuration. I think we would benefit from cleaning that up and have redirects implemented with native solutions like https://documatt.gitlab.io/sphinx-reredirects/

Reply:
https://github.com/conda/conda/pull/12798/commits/3bd80d0c26718289734a2291175761661f3d0c42